include/pipettes/core/tasks/i2c_task.hpp

```diff
@@ -35,6 +35,17 @@ class I2CMessageHandler {
                                    TIMEOUT);
         m.client_callback(m.buffer);
     }
+
+    void visit(PollReadFromI2C &m) {
+        auto empty_array = m.buffer;
+        for (int i = 0; i < m.polling; i++) {
+            i2c_device.central_receive(m.buffer.data(), m.buffer.size(),
```

Comment:
We should make sure there's a wait or a yield or something somewhere in this loop so we don't busywait the entire cpu. it's also a bit weird to me that the timing isn't defined and isn't saved anywhere, but I don't know what this is for so maybe it's fine.

Reply:
Yep that was an oversight on my side.